examples/sim/README.md

```diff
@@ -0,0 +1,29 @@
+# Ibex tracer simulation example
+
+## Overview
+
+This examples shows the usage of the module `ibex_core_tracer` which forwards
+all port signals to the `ibex_core` and a subset of signals to `ibex_tracer`.
+The tracer will create a file with a stream of executed instructions.
+
+## Prerequisites
+
+For this example, `modelsim` must be available and the following environment
+variable must point to the path of installation:
+
+```
+export MODEL_TECH=/path/to/modelsim/bin
+```
+
+## Usage
+
+In the `examples` directory of `ibex` run the following command:
+
+```
+fusesoc --cores-root=.. run --target=sim lowrisc:ibex:top_tracer_sim
```

Comment:
We typically recommend running fusesoc from the toplevel, which would make the comment this:
```
fusesoc --cores-root=. run --target=sim lowrisc:ibex:top_tracer_sim
```